autoscale-kubernetes-container/example-dev-setup/autoscaler.yaml

```
@@ -0,0 +1,30 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: autoscaler-deployment
+  labels:
+    app: autoscaler
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: autoscaler
+  template:
+    metadata:
+      labels:
+        app: autoscaler
+    spec:
+      containers:
+        - name: autoscaler
+          image: dev/autoscaler/autoscale-kubernetes-rabbit:1.9.0-SNAPSHOT
```

Perhaps we should reference the latest released image so that we won't have to maintain this version number?

```suggestion
          image: autoscaler/autoscale-kubernetes-rabbit:latest
```

---

Done